packages/integration-tests/src/resolvers/entityResolver.ts

```diff
@@ -0,0 +1,3 @@
+import { Entity, EntityMetadata } from "joist-orm";
+
+export function entityResolver<T extends Entity>(entityMetadata: EntityMetadata<T>): any {}
```

Comment:
Should have taken the chance to change the signature of this to just take an `EntityConstructor` constructor directly.  It's just added boilerplate to take an `EntityMetadata` that's even hardwired into the codegen now.

Reply:
Also sort of confused how consumers that aren't us would actually get an `entityResolver` or understand what it is for.  Seems like we should at least have a default implementation if we're going to be codegenning it?

Reply:
Yeah, so none of these `entityResolver` / etc. utilities are meant for consumption / usage / anything more than "just make sure the currently-generated code compiles for the purposes of the integration test".

I'm lazy punting extracting the utilities "for real" to a future PR / probably some sort of new `joist-graphql-utils` package.

Reply:
And yeah agreed we should change to `EntityConstructor` when we do that.